src/components/ShareButtons/index.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ReactTooltip from 'react-tooltip';
+import {
+  EmailShareButton,
+  FacebookShareButton,
+  InstapaperShareButton,
+  LineShareButton,
+  LinkedinShareButton,
+  PocketShareButton,
+  RedditShareButton,
+  TwitterShareButton,
+  WhatsappShareButton,
+
+  EmailIcon,
+  FacebookIcon,
+  InstapaperIcon,
+  LineIcon,
+  LinkedinIcon,
+  PocketIcon,
+  RedditIcon,
+  TwitterIcon,
+  WhatsappIcon
+} from "react-share";
+
+import ShareIcon from './ShareIcon';
+
+const ShareButtons = ({
+  description,
+  title,
+  postUrl
+}) => {
+
+  const [showTooltip, setShowTooltip] = useState(true);
```

Comment:
nit: is prettier working for you? it should run on commit.. we have `semi: false` in `.prettierrc` but it looks like semi colons are making it in, not that I care strongly about whether or not semicolons are used but I thought prettier should've run

Reply:
i don't think it happened on commit - should it prevent me from committing?

Reply:
no, it should just run `lint-staged` (in the package.json) when you try to commit. maybe `git pull --rebase` this branch because you have conflicts (probably `blog-post.js`'s placement of the language switcher from #3) and maybe it'll run when you fix the merge conflicts? otherwise you can `npx prettier --write src/**/*.js` to get a manual prettier run...

or we can just not bother and the next time the file is touched it'll probably be prettier'd